p4-16/spec/P4-16-spec.mdk

```diff
@@ -7683,6 +7683,7 @@ syntactic names:
 | Constants. | `1` | `"1"` |
 | Field projections. | `data.f1` | `"data.f1"` |
 | Slices. | `f1[3:0]` | `"f1[3:0]"` |
+| Masks. | `h.src & 0xFFFF` | `"h.sec & 0xFFFF"` |
```

Comment:
typo in the generated field name

Reply:
we could make these just suggestions, and not mandatory rules for all implementations.